github-email-notifications/config.py

```diff
@@ -1 +1,16 @@
-config = dict(githubconsumer=True)
+config = {
+    "testgithubconsumer": False,  # set to True for enabling consumer
+    "testgithubconsumer.email_to": u"test@example.com",
+    "testgithubconsumer.email_from": u"noreply@example.com",
+    "testgithubconsumer.smtp_server": u"smtp.example.com",
+
+    "sssdgithubconsumer": False,
+    "sssdgithubconsumer.email_to": u"test@example.com",
+    "sssdgithubconsumer.email_from": u"noreply@example.com",
```

Comment:
This is config file, we can put there anything we want, I just did not want to set publicly any internal servers. This has not been discussed yet, but it is easy to change.